### PR TITLE
feat: support SKIP_INSTALL_SIMPLE_GIT_HOOKS env

### LINKED ISSUE #99: [feat] Add a new option to skip enabling on CI
As title

### PR DESCRIPTION
**Pull request by JounQin**
close #99

### CODE REVIEW

**Comment by toplenboren** [comment]
Hey! Thanks for the PR

Can you please double check yarn.lock? Its strange that the list of dependencies is not changed, but yarn.lock is changed

File: README.md [hunk 1]
```diff
@@ -162,7 +162,15 @@ npm uninstall simple-git-hooks
 
 ### I want to skip git hooks!
 
-You should use `--no-verify` option
+If you need to bypass install hooks at all, for example on CI, you can use `SKIP_INSTALL_SIMPLE_GIT_HOOKS` environment variable at the first place.
```
Comment (toplenboren) [inline README.md:165]: Do we need to handle delete simple git hooks in some way? Like, what will happen when we have `SKIP_INSTALL_SIMPLE_GIT_HOOKS=1` and we try to delete hooks?

--

wyt?

**Comment by JounQin** [comment]
@toplenboren It seems `yarn.lock` not been updated for quite a long time.

File: README.md (hunk 1 above)
Comment (JounQin) [reply README.md:165]: I don't quite to follow, `SKIP_INSTALL_SIMPLE_GIT_HOOKS=1` means no hooks to be installed at all, what to delete then?

**Review by toplenboren — APPROVED** [review]
lgtm

File: cli.js [hunk 2]
```diff
@@ -6,6 +6,13 @@
  */
 const {setHooksFromConfig} = require('./simple-git-hooks')
 
+const {SKIP_INSTALL_SIMPLE_GIT_HOOKS} = process.env
+
+if (['1', 'true'].includes(SKIP_INSTALL_SIMPLE_GIT_HOOKS)) {
+    console.log(`[INFO] SKIP_INSTALL_SIMPLE_GIT_HOOKS is set to "${SKIP_INSTALL_SIMPLE_GIT_HOOKS}", skipping installing hook.`)
```
Comment (JounQin) [inline cli.js:12]:
```suggestion
    console.log(`[INFO] SKIP_INSTALL_SIMPLE_GIT_HOOKS is set to "${SKIP_INSTALL_SIMPLE_GIT_HOOKS}", skipping installing hook, if you had hooks installed, you can delete them using the delete command.`)
```

@toplenboren Did you mean this? What is `delete command` by the way?

Comment (toplenboren) [reply cli.js:12]: I meant this:

image

You can run `npm uninstall simple-git-hooks` and we will trigger `removeHooks()` or smth like that

Comment (toplenboren) [reply cli.js:12]: But that will prompt user to uninstall the package .... lets just leave it as is

Sorry for inconvinience

**Comment by JounQin** [comment]
So, it's ready to merge.

**Pull request merged**